ios/FluentUI/List/ListItem.swift

```diff
@@ -274,6 +275,7 @@ public struct ListItem<LeadingContent: View,
     // MARK: Private variables
 
     @Environment(\.fluentTheme) private var fluentTheme: FluentTheme
+    @Environment(\.isEnabled) private var isEnabled: Bool
```

Comment:
Hopefully dumb question - why does this need to be an Environment variable? Feels like it should be specific to the view?

Reply:
it is set based on the `.disabled()` view modifier. This allows for ListItem to handle changing on disabled state without having to have a custom modifier to pass in a boolean value. 

It is an environment variable as that allows it to be set without having each view have to pass the value down. So for example, if a whole list is disabled, it will allow for this value set and used without having to pass in a boolean to each listItem view.